test/scenarios/17. settings/advanced/show-test-networks.csv

```diff
@@ -0,0 +1,16 @@
+Advanced settings: Check “Show test networks” toggle works correctly,,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots),,Drop down menu should appear,
+4,Click on the Settings from menu.,,Settings page appears,
+5,Click on the Advanced tab.,,Advanced settings page appears,
+6,Validate that the 'Show test networks' is default to off.,,The toggle is set to off by default.,
+7,Click on the Network selection drop down on the left top,,Select a network dialog box appears,
+8,Validate that the 'Show test networks' toggle,,The 'Show test networks' toggle is off,
+9,Close the 'Select test networks' dialog,,Settings page should appear,
+10,Toggle the 'Show test networks' switch.,,The switch is turned on as expected,
+11,Click on the Network selection drop down on the left top,,Select a network dialog box appears,
+12,Validate that the 'Show test networks' toggle is ON,,The 'Show test networks' toggle is selected ON automatically,
+13,Validate that the test networks appear,,The test networks is shown as expected,
+14,Toggle the 'Show test networks' switch in the select network dialog.,,The switch is turned off as expected in the settings page,
```

Comment:
This seems like it may be multiple steps.  Toggle the setting in network dialog, close network dialog to land back in settings to assert.  Also, same feedback on the action as line 12/step 10, but in this case "Toggle....Off"

Reply:
Thanks Mike. I will need to verify this step with the relevant side of the application and will provide updates if there are any additional steps.

Reply:
I changed the format of the test scenario and its fixed.